### ISSUE TITLE
ContainerSSH does not work with vscode remote ssh

### ISSUE THREAD

**Issue by RaananHadar** [ContainerSSH/ContainerSSH#485]
## Describe the bug
[VSCode remote SSH](https://code.visualstudio.com/docs/remote/ssh) is a very common way to remotely develop via ssh on virtual machines. I successfully deployed the kubernetes quickstart and connected via a standard ssh client. However vscode remote ssh fails to work for some reason. Given that it does work with normal vm ssh sessions, It is my understanding that something with the ssh implementation in ContainerSSH is amiss and not vscode.

## To Reproduce
Deploy the quickstart on k8s and attempt to connect via vscode remote ssh.

## Expected behavior
VSCode should work with ContainerSSH

## Version
0.4.1

**Comment by janosdebugs** [comment]
Hello @RaananHadar thank you very much for this issue. Can you give us your configuration that you used with k8s? Did you use the example, or do you have a custom configuration?

**Comment by janosdebugs** [comment]
Hello @RaananHadar thank you very much, we will try to reproduce this and see if we can fix the issue.

**Comment by janosdebugs** [comment]
Hey @RaananHadar I believe I have found the issue: VSCode remote is requesting TCP/IP forwarding into the environment, which is currently not supported by ContainerSSH. If the SSH server doesn't support this, VSCode simply hangs and doesn't open the remote connection. You can see this in the logs by typing `kubectl logs -n containerssh containerssh-INSERT-POD-ID containerssh`:

```json
{"timestamp":"2022-05-31T10:43:11Z","level":"debug","code":"SSH_UNSUPPORTED_CHANNEL_TYPE","message":"Unsupported channel type requested","details":{"channelId":6,"connectionId":"06f8427040bd419e976cb86bcb0f91fc","module":"ssh","remoteAddr":"10.244.0.1","type":"direct-tcpip","username":"foo"}}
```

TCP/IP support is being implemented in ContainerSSH version 0.5 by @tsipinakis in containerssh/libcontainerssh#103. Please track that issue for support.

Side note: the example deployment uses the `LoadBalancer` service type. This does not work in Kubernetes deployments that don't have a load balancer, such as KinD. You will need to switch to `NodePort` for such setups.

**Please feel free to reopen this issue if you need additional help** or join the ContainerSSH Slack channel for further help.
